src/client/components/pages/collection.js

```diff
@@ -224,6 +232,59 @@ class CollectionPage extends React.Component {
 		}, this.pagerElementRef.triggerSearch);
 	}
 
+	handleSubscribe() {
+		const submissionUrl = '/subscription/subscribe/collection';
+		const collectionId = this.props.collection.id;
+		const subscriberId = this.props.userId;
```

Comment:
Would this not be accessible in the route handler on the server?
How come you don't need to send the userId for the entity subscribe routes but you do send them for the collections subscribe route?